packages/manager/apps/catalog-revamp/package.json

```diff
@@ -31,13 +31,16 @@
     "@tanstack/react-query-devtools": "^4.9.0",
     "bootstrap": "^5.3.0",
     "clsx": "^1.2.1",
+    "lodash": "^4.17.21",
```

Comment:
Is lodash really really required? We can use native JS functions.

Reply:
It was used in the original Angular JS code, let me check to do it with native JS functions

Reply:
Done